packages/react/src/patterns/sections/CardSection/CardSectionSimple.js

```diff
@@ -1,25 +1,26 @@
 import CardSection from './CardSection';
 import PropTypes from 'prop-types';
 import React from 'react';
-
 /**
- * SimpleCards pattern it is Cards without images
+ * CardSectionSimple pattern it is Cards without images
  *
  * @param {object} props props object
  * @param {Array} props.cards cards array with title, and cards properties
  * @returns {object} JSX Object
  */
-const SimpleCards = props => {
-  return <CardSection {...props} />;
+const CardSectionSimple = props => {
+  const cardWithoutImages = props.cards.filter(
+    card => !card.image && !card.eyebrow
+  );
+  return <CardSection {...(cardWithoutImages && { ...props })} />;
```

Comment:
I think I see what you're trying to do here, but I don't think we need to overcomplicate it here. If we're doing a check for whether the card data is there or not it should be done in CardSection. Would everyone flip out if we just add the filtered cards back to the prop?

You also need to explicitly check for the other required props. Even the ones you don't want.

```js
const CardSectionImages = props => {
  props.cards = props.cards.filter(
    {image, eyebrow, heading, copy, cta: { icon, href }}) =>
     !image && !eyebrow && heading && copy && cta && icon && href 
  );
  return <CardSection {...props} />;
}
```